boards/arm/nrf5340dk_nrf5340/CMakeLists.txt

```diff
@@ -70,6 +70,13 @@ if (CONFIG_BUILD_WITH_TFM)
 				$<TARGET_PROPERTY:tfm,TFM_S_BIN_FILE>
 				${CMAKE_BINARY_DIR}/tfm_s_signed.bin
 
+			# Generate an intel hex file from the signed output binary
+			COMMAND srec_cat ${CMAKE_BINARY_DIR}/tfm_s_signed.bin
```

Comment:
Never call a command directly, always use `find_program()` and ensure this code is not activated if `srec_cat` is missing.

It's already looked up here: https://github.com/nrfconnect/sdk-zephyr/blob/21046b8cdb4eac989c6b17bb21ffc8196be3d5e4/boards/arm/nrf5340dk_nrf5340/CMakeLists.txt#L40

Reply:
Fixed